### DISCUSSION TITLE
Use Geofence Server from master in node when clustering

### DISCUSSION THREAD

**Discussion by g1ku-source** [kartoza/docker-geoserver#649] [Q&A]
Hello! I am using the clustering docker compose project from the develop branch and I installed the Geofence server extension in the master container and the geofence extension in the node container. However, I cannot make the client node use the Geofence server from the master. Did I understand something wrong about how I can use the Geofence server?

**Comment by NyakudyaA** [comment]
Hi, what is your use case of Geofence?

**Reply by g1ku-source** [reply]
I want to use it in a cluster of geoservers where the changes are made directly to master and the clients can use the configuration from the master itself rather than transmitting the changed rule across the entire cluster.
